sdk/redis/Azure.ResourceManager.Redis/src/autorest.md

```diff
@@ -20,6 +20,7 @@ modelerfour:
   lenient-model-deduplication: true
 use-model-reader-writer: true
 enable-bicep-serialization: true
+tag: package-2024-03
```

Comment:
is the default tag on the readme.md with above commit id also this `package-2024-03`?
If so we could remove this line.
Usually we do not add this if the default tag has the same value, because this kind of disallows the SDK automation pipeline to automatically update the SDK

Reply:
No unfortunately the default tag is package-preview-2024-04 right now, which we don't want to release as part of the SDK. This soon will be fixed as part of an upcoming release